README.md

```diff
@@ -2,37 +2,34 @@
 
 The dotnet/core repository is a good starting point for .NET Core. 
 
-## Get Started
-
-If you're new to .NET Core, you can get started in 10 minutes: 
+## Learn about .NET Core 
 
-- [Get started with .NET Core](https://www.microsoft.com/net/core)
-- [Download .NET Core](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
+* [Learn about .NET Core](https://docs.microsoft.com/dotnet/core)
+* [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md)
+* [Learn how about the .NET platform](https://docs.microsoft.com/dotnet/standard/)
 
-## Learn about .NET Core 
+## Download .NET Core
 
-- [Learn about .NET Core](https://docs.microsoft.com/dotnet/core)
-- [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md)
-- [.NET Core release notes](https://github.com/dotnet/core/blob/master/release-notes/README.md)
-- [Learn how about the .NET platform](https://docs.microsoft.com/dotnet/standard/)
+* [Released builds](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
+* [.NET Core release notes](https://github.com/dotnet/core/blob/master/release-notes/README.md)
+* .NET Core 2.0 daily builds: 
```

Comment:
I was figuring that fell under release builds. I guess your point is that we also have daily 1.1 builds since we're actively servicing that product. Yes? I was thinking more about previews and less about the servicing scenario. That's a good thought. Is that your point?